admin/build-doc

```diff
@@ -22,7 +22,11 @@ if command -v dpkg >/dev/null; then
 elif command -v yum >/dev/null; then
     for package in python-devel python-pip python-virtualenv doxygen ditaa ant libxml2-devel libxslt-devel Cython graphviz; do
 	if ! rpm -q $package >/dev/null ; then
-		missing="${missing:+$missing }$package"
+		command -v `echo $package | tr '[:upper:]' '[:lower:]' | awk -F "-" '{ if ($2 == "") print $1; else print $2}'`   > /dev/null;
```

Comment:
this doesn't quite work for me, could you give examples on what input/output this works with? 

This approach is brittle, and breaks on some packages:

    $ rpm -q ant | tr '[:upper:]' '[:lower:]' | awk -F "-" '{ if ($2 == "") print $1; else print $2}'
    1.9.2

Reply:
@alfredodeza Yes, I am using multi-pipe with `tr` and `awk` to verify if the executable exists. 
It is just adding an extra cushion before returning failure, even if the package is installed. And it will not affect the existing checks because it is preventing incorrect errors.

>I think this is still a brittle way to verify this. There is more than one package that doesn't have an executable, 

I agree that there can be various packages without any executable, however if the package tests passes, it will not at all check for executable.

>or that the executable that your pipe line is trying to parse is incorrect (e.g. python-devel would parse as 'devel').

If `python-devel` package is present, it will not parse the `tr` and `awk` line of code.
And if `python-devel` is missing it will give the error anyhow. 

>The problem here is not that the executable may be there and we should assume then that the package is installed, the problem here is that you need to have the right packages for FC25, which this script doesn't account for.

But aren't we expecting the executable as of importance than package ?
I mean, if I have the executable from latest version of some package, why it should fail ?